src/ui/elements/job-view-record.element.ts

```diff
@@ -23,25 +25,20 @@ export const JobViewRecord = defineElement<{
 
         const entryTitle = inputs.record.contactName || inputs.record.companyName || dateString;
 
-        const tableRows = getObjectTypedEntries(inputs.record)
-            .sort(([a], [b]) => a.localeCompare(b))
-            .map(
-                ([
-                    recordKey,
-                    recordValue,
-                ]) => {
-                    if (!recordValue || !check.isString(recordValue)) {
-                        return nothing;
-                    }
+        const jobRecordKeyOrder: (keyof JobSearchRecord)[] = getObjectTypedKeys(
+            jobSearchRecordShape.shape,
+        );
 
-                    return html`
-                        <tr>
-                            <th>${jobSearchRecordPropertyDisplayNames[recordKey]}:</th>
-                            <td>${recordValue}</td>
-                        </tr>
-                    `;
-                },
-            );
+        const tableRows = jobRecordKeyOrder.map((recordKey) => {
+            if (recordKey === 'contactDate') return nothing;
```

Comment:
this used to omit empty values as well, lets keep doing that:
```suggestion
            if (recordKey === 'contactDate' || !inputs.record[recordKey]) return nothing;
```

Reply:
Done!